cloud-aws-common/build.gradle

```diff
@@ -21,6 +21,9 @@ dependencies {
   implementation project(':common-handlebar')
   implementation project(':template-manager-tag')
   implementation project(':auth-connector')
+  implementation project(':usage-collection')
```

Comment:
I dont think we need this because cloud api already using it

Reply:
Actually we need it because there is implementation of convert function in "aws-common" module and it uses MeteringEventsProto from "usage-collection" module.